test/runtime/pod.go

```diff
@@ -107,6 +107,31 @@ func (builder *PodBuilder) AddConfigMapVolume(name, configMapName string) *PodBu
 	return builder
 }
 
+func (builder *PodBuilder) AddSecretVolume(name, secretName string) *PodBuilder {
+	builder.Pod.Spec.Volumes = append(builder.Pod.Spec.Volumes, corev1.Volume{
+		Name: name,
+		VolumeSource: corev1.VolumeSource{
+			Secret: &corev1.SecretVolumeSource{
+				SecretName: secretName,
+			},
+		},
+	})
+	return builder
+}
+
+func (builder *PodBuilder) GetContainer(name string) *ContainerBuilder {
```

Comment:
I'm asking but don't know the answer.  Is there any other way to get a container? Does it makes sense to name it "GetContainerByName" ?

Reply:
Not really. The name is the only attribute of a container that's required to be unique in a Pod, so we will need to return a collection of containers if we look up by any other attribute.  https://github.com/kubernetes/api/blob/5c8efac7a372214f8b0673cb92073d73f3881818/core/v1/types.go#L2229